tests/test_ess_csc.py

```diff
@@ -228,6 +228,9 @@ async def test_rpi_data_client_loses_connecton(self) -> None:
             fault = await self.remote.evt_errorCode.next(
                 flush=False, timeout=STD_TIMEOUT
             )
+            fault = await self.remote.evt_errorCode.next(
+                flush=False, timeout=STD_TIMEOUT
+            )
```

Comment:
Why have the same code here? Any reason?

Reply:
The first time the error code received is not the expected one. The second time it is. I reported this to Russell, who implemented the test and asked him to investigate.